accel-pppd/memdebug.c

```diff
@@ -80,12 +80,12 @@ void __export md_free(void *ptr, const char *fname, int line)
 	mem = container_of(ptr, typeof(*mem), data);
 
 	if (mem->magic1 != MAGIC1) {
-		printf("memory corruption:\nfree at %s:%i\n", fname, line);
+		printf("md_free: memory corruption:\nfree at %s:%i\n", fname, line);
```

Comment:
we've talked about it already, better to use __func__ macro instead of hardcoded values